src/Plugins/OrientationAnalysis/src/OrientationAnalysis/Filters/Algorithms/GenerateFaceIPFColoring.cpp

```diff
@@ -185,6 +185,7 @@ Result<> GenerateFaceIPFColoring::operator()()
 
   ParallelDataAlgorithm parallelTask;
   parallelTask.setRange(0, numTriangles);
+  parallelTask.setParallelizationEnabled(false);
```

Comment:
Check if parallelization disabled is intended.

Reply:
@mmarineBlueQuartz Can we check if the DataArray is out-of-core then turn off parallel otherwise leave it on?